polaris-react/src/components/Banner/Banner.scss

```
@@ -31,28 +50,82 @@
   &.statusSuccess {
     border-color: var(--p-color-border-success-subdued);
     background-color: var(--p-color-bg-success-subdued);
+
+    #{$se23} & {
+      border-color: none;
+      background-color: none;
+      @if $in-page == false {
+        color: var(--p-color-text-success);
+      }
+    }
   }
 
   &.statusInfo {
     border-color: var(--p-color-border-info-subdued);
     background-color: var(--p-color-bg-info-subdued);
+
+    #{$se23} & {
+      border-color: none;
+      background-color: none;
+      @if $in-page == false {
+        color: var(--p-color-text-info);
+      }
+    }
   }
 
   &.statusWarning {
     border-color: var(--p-color-border-caution-subdued);
     background-color: var(--p-color-bg-caution-subdued);
+
+    #{$se23} & {
+      border-color: none;
+      background-color: none;
+      @if $in-page == false {
+        color: var(--p-color-text-warning);
+      }
+    }
   }
 
   &.statusCritical {
     border-color: var(--p-color-border-critical-subdued);
     background-color: var(--p-color-bg-critical-subdued);
+
+    #{$se23} & {
+      border-color: none;
+      background-color: none;
+      @if $in-page == false {
+        color: var(--p-color-text-critical);
+      }
+    }
   }
 }
 
 .Banner {
   // stylelint-disable -- Polaris component custom properties
   --pc-banner-secondary-action-horizontal-padding: var(--p-space-3);
   --pc-banner-secondary-action-vertical-padding: var(--p-space-2);
+
+  :where(#{$se23} &) {
```

The `:where` is now included in the global selector so I'm hoping we can just do:
```suggestion
  #{$se23} & {
```